kedro_kubeflow/auth.py

```diff
@@ -4,6 +4,9 @@
 from urllib.parse import urlsplit, urlunsplit
 
 import requests
+from google.auth.exceptions import DefaultCredentialsError
```

Comment:
I'm not sure about it - in general, I see that google auth is in the requirments, but in long-term it should be optional, as we can use this plugin to run pipelines on AWS/Azure too.